ios_automation/page_action/like_page.py

```diff
@@ -0,0 +1,269 @@
+from appium.webdriver.common.appiumby import AppiumBy
+from selenium.common import NoSuchElementException
+
+import com_utils.element_control
+
+
+def close_brand_recommended_page(wd):
+    try:
+        wd.find_element(AppiumBy.ACCESSIBILITY_ID, 'recommended_brand_page')
+        wd.find_element(AppiumBy.ACCESSIBILITY_ID, 'icNavigationbarBackBlack').click()
+        print('브랜드 추천 페이지 노출')
+    except NoSuchElementException:
+        pass
+
+
+def set_like_zero(self, wd):
+    # 좋아요 api 호출하여 각 탭의 좋아요 수 확인
+    my_heart = com_utils.api_control.my_heart_count(self.pconf['id_29cm'], self.pconf['password_29cm'])
+    product_count = my_heart['product_count']
+    brand_count = my_heart['brand_count']
+    post_count = my_heart['post_count']
+
+    if product_count != 0:
+        print(f'좋아요 상품 수 : {product_count}')
+        # product 탭 진입하여 모든 상품 좋아요 해제
+        click_product_tab(wd)
+        click_to_unlike_product(wd)
+        # 좋아요 해제 후 새로고침
+        refresh_product_like_tab(wd)
+
+    if brand_count != 0:
+        print(f'좋아요 브랜드 수 : {brand_count}')
+        click_brand_tab(wd)
+        click_to_unlike_brand(wd)
+        # 좋아요 해제 후 새로고침
+        refresh_brand_like_tab(wd)
+
+    if post_count != 0:
+        print(f'좋아요 포스트 수 : {post_count}')
+        click_post_tab(wd)
+        click_to_unlike_post(wd)
+        # 좋아요 해제 후 새로고침
+        refresh_post_like_tab(wd)
+
+
+# 비교하는 like 수를 like_count에 작성
+def check_like_total_count(wd, warning_texts, like_count):
+    like_total_count = com_utils.element_control.ial(wd, 'like_total_count')
+    if like_total_count.text == like_count:
+        test_result = 'PASS'
+        print('총 LIKE 개수 확인')
+    else:
+        test_result = 'WARN'
+        warning_texts.append('총 LIKE 개수 확인 실패')
+        print('총 LIKE 개수 확인 실패')
+    return test_result
+
+
+def click_product_tab(wd):
+    com_utils.element_control.ialc(wd, 'like_product_tab')
+
+
+def click_brand_tab(wd):
+    com_utils.element_control.ialc(wd, 'like_brand_tab')
+
+
+def click_post_tab(wd):
+    com_utils.element_control.ialc(wd, 'like_post_tab')
+
+
+def check_no_product_like(wd, warning_texts):
+    try:
+        com_utils.element_control.ial(wd, '좋아요한 상품이 없습니다. 마음에 드는 상품의 하트를 눌러보세요.')
```

Comment:
from com_utils.element_control import ial, ialc, ialk
를 하면 
```suggestion
        ial(wd, '좋아요한 상품이 없습니다. 마음에 드는 상품의 하트를 눌러보세요.')
```
이렇게 할 수 있습니다. 이게 진짜 짧게 쓰기 위한 의도이기도 하구요~

Reply:
넵! 코드 수정하겠습니다.